base/sim/FairMCApplication.h

```diff
@@ -302,13 +303,15 @@ class FairMCApplication : public TVirtualMCApplication
     /** Current state */
     FairMCApplicationState fState;   //!
 
-    ClassDef(FairMCApplication, 4);
+    ClassDefOverride(FairMCApplication, 4);
```

Comment:
Since you changed a data member, you should increase the version counter.

Reply:
I think, I only changed `fRadGridMan`.
This one was and still is marked with `//!`.

From looking at [Chappter 13.3.2 Transient Data Members](https://root.cern/root/htmldoc/guides/users-guide/InputOutput.html#transient-data-members) it seems that this marks a member variable as not taking part in marshalling.

I had assumed, that the version marking is resposible for on-disk versions?

Reply:
You are right. Normally I increase the version when I change something but in this case it shouldn't be necessary.